Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -453,16 +457,14 @@ protected class ClassWriter : IClassWriter {
     }
 
     protected override ConcreteSyntaxTree CreateLabeledCode(string label, bool createContinueLabel, ConcreteSyntaxTree wr) {
-      if (createContinueLabel) { throw new NotImplementedException(); }
+      Contract.Assert(!createContinueLabel);
```

Comment:
This was suggested in review. Let's keep it that way.

Reply:
so just to be sure are you suggesting to keep the line 456 as first line of the function, right ?

Reply:
I think so, just don't change anything.